pom.xml

```diff
@@ -576,6 +581,11 @@
         <version>${jersey.version}</version>
       </dependency>
       <dependency>
+        <groupId>org.glassfish.jersey.media</groupId>
+        <artifactId>jersey-media-jaxb</artifactId>
+        <version>${jersey.version}</version>
+      </dependency>
```

Comment:
If you are relying on the dependency to be brought in transitively it is better to explicitly state them. cc @apc999